src/components/com_tjcertificate/administrator/models/certificates.php

```diff
@@ -130,11 +131,15 @@ protected function getListQuery()
 		}
 
 		// Filter by user id
-		$userId = $this->getState('filter.user_id');
 
-		if (!empty($userId))
+		if (!$user->authorise('core.admin'))
 		{
-			$query->where($db->quoteName('ci.user_id') . ' = ' . (int) $userId);
+			$userId = $this->getState('filter.user_id');
+
+			if (!empty($userId))
+			{
+				$query->where($db->quoteName('ci.user_id') . ' = ' . (int) $userId);
+			}
```

Comment:
Show demo for this scenario

Reply:
old behaviour is admin ubable to see all certificate so I am allowing admin to see all added certificates